nautobot/docs/configuration/optional-settings.md

```diff
@@ -465,6 +465,8 @@ Please see [the object permissions page](../administration/permissions.md) for m
 
 Default: `None`
 
+Environment Variable: `SCRIPT_NAME`
```

Comment:
Please verify this one, I believe it is correct, but work another look.

Reply:
I'm not an expert on this feature but I don't think this is correct. If I'm reading the Django docs correctly, the idea is that if `$SCRIPT_NAME` is set in your environment / on your HTTP server, you must *also* set `settings.FORCE_SCRIPT_NAME` to match that value.

Reply:
Honestly, I still don't get it :) 

The way it is worded originally, I am not sure what is being requested. Would you mind making a suggestion to what you think is correct?

Reply:
Yeah this is a bit confusing. This is a server internal environment variable